### DISCUSSION TITLE
journal/2024-02-18-2023nian-du-zong-jie/

### DISCUSSION THREAD

**Discussion by giscus** [nnnewb/nnnewb.github.io#3] [Announcements]
# journal/2024-02-18-2023nian-du-zong-jie/

弃船！

https://nnnewb.github.io/journal/2024-02-18-2023nian-du-zong-jie/

**Comment by schunlee** [comment]
居然github page搭的网站也备案了，真是好孩子。2024加油。

**Reply by nnnewb** [reply]
谢谢第一条评论，单机好久了；）
